pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -1604,7 +1604,15 @@ void setDelayedDeliveryPolicy(String topic
      * @throws PulsarAdminException
      *             Unexpected error
      */
-    int getMessageTTL(String topic) throws PulsarAdminException;
+    Integer getMessageTTL(String topic) throws PulsarAdminException;
+
+    /**
+     * Get message TTL applied for a topic.
+     * @param topic
+     * @return
+     * @throws PulsarAdminException
+     */
+    int getMessageTTLApplied(String topic) throws PulsarAdminException;
```

Comment:
```suggestion
    Integer getMessageTTL(String topic, boolean applied) throws PulsarAdminException;
```

Reply:
How about we set `int getMessageTTLApplied(String topic)` to @Deprecated, and add a new `Integer getMessageTTL(String topic, boolean applied)`?